webgl/main.js

```diff
@@ -2,6 +2,9 @@ const {app, BrowserWindow} = require('electron');
 
 let mainWindow;
 
+// enable ignore-gpu-blacklist option
```

Comment:
I'm fine with adding this option in the sample, but please add comment stating that this option would make electron/chromium less stable -- chromium blacklists some gpus because of some bugs of these gpus.

Reply:
@hokein Added comments. Let me know if its ok! Thanks 😄